server/docs/api.md

```diff
@@ -60,6 +69,37 @@ $ curl -H "Accept: application/json" https://bookworms-api.vercel.app/api/review
 }
 ```
 
+If you would like to return sorted result according to populality likes
+
+```
+$ curl -H "Accept: application/json" https://bookworms-api.vercel.app/api/reviews?sort=1
+
+```
+
+This will return data in ascending order based on likes.
+
+```
+$ curl -H "Accept: application/json" https://bookworms-api.vercel.app/api/reviews?sort=-1
+
+```
+
+To enable search you need to todo it this way
+
+```
+$ curl -H "Accept: application/json" https://bookworms-api.vercel.app/api/reviews?search=Keyword here
+
+```
+
+To do sort and search at the sametime do
+
+```
+$ curl -H "Accept: application/json" https://bookworms-api.vercel.app/api/reviews?search=Keyword here&sort=1
```

Comment:
What if I want to sort, search and filter at the same time?

Reply:
So if I want to search for `Harry Potter` in `Magic` category and sort by recency, do I need to call 2 APIs?

Reply:
@veritem It would be great you can help to answer this question before we merge this PR.

Reply:
Oh, no need for this now as I see your update in the code alr. Glad that you update it. :smile: